README.md

```diff
@@ -1,34 +1,110 @@
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 [![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/960/badge)](https://bestpractices.coreinfrastructure.org/projects/960)
+[![Codacy Badge](https://api.codacy.com/project/badge/Grade/4d8edb74d4954c76a4656a9e109dbc4e)](https://www.codacy.com/app/neewy/iroha?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=hyperledger/iroha&amp;utm_campaign=Badge_Grade)
+[![CircleCI](https://circleci.com/gh/hyperledger/iroha/tree/master.svg?style=svg)](https://circleci.com/gh/hyperledger/iroha/tree/master)
+
+Blockchain platform Hyperledger Iroha is designed for simple creation and management of assets. This is a distributed ledger of transactions.
+
+<img height="300px" src="docs/Iroha_3_sm.png"
+ alt="Iroha logo" title="Iroha" align="right" />
+
+It does following things:
+
+Among the basic functional requirements of the system are the following groups:
+1. Asset management: where asset could be the equivalent of a currency or as indivisible assets — land rights, serial numbers, patents, etc.
+2. Management of user accounts, user domains
+3. The system of rights and verification of user permissions for the execution of transactions and requests in the system
+4. Validation of business rules for transaction in the system
+
+
+Among the non-functional requirements can be noted:
+* the ability to deploy the system in linux containers on a standard configuration of office computers (no more than 2GB of RAM, 1TB storage)
+* high degree of network fault tolerance (the system continues its operability and guarantees data reliability provided that no more than F nodes of the network fail at the same time, where 3F + 1 is the total number of nodes)
+* quick processing of queries (less than 3 seconds in normal operating conditions)
+* the system's bandwidth capacity (expected value of 100 thousand transactions per second)
+
+## Current Status
+
+Iroha v0.95 preview veriosn is released on [TBD].
+
+## Iroha repository 101
+
+System runs as a daemon, representing single peer in Iroha network. For each peer there is following package structure, corresponing to components of the system:
+
+<img height="300px" src="docs/simple-package-diagram.png"
+ alt="Diagram" title="Diagram" align="right" />
+
+* Torii (gate) — package contains classes, which are in charge of interaction with users (clients)
+* Network — interaction in network of peers
+* Validation — checks business rules and validity (right format) of transactions or queries
+* Synchronizer — helps to sync newcomers or temporary disconnected peers
+* Simulator — generates temporary snapshot of storage to validate transactions
+* Ametsuchi — ledger block storage
+* Model — system entities, and converters for them
+
+<br>
+
+There components are passing following _model objects_, which represent entities in the system:
```

Comment:
Is this information really required for main page? Looks like many definitions without purpose.